3rdParty/SDL_audiolib/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+include(FetchContent_MakeAvailableExcludeFromAll)
+
+if(DEVILUTIONX_STATIC_SDL_AUDIOLIB)
+  set(BUILD_SHARED_LIBS OFF)
+else()
+  set(BUILD_SHARED_LIBS ON)
+endif()
+
+# No need for the libsamplerate resampler:
+set(USE_RESAMP_SRC OFF)
+
+# No need for the SOX resampler:
+set(USE_RESAMP_SOXR OFF)
+
+# We do not need any of the audio formats except WAV:
+set(USE_DEC_DRWAV ON)
+set(USE_DEC_DRFLAC OFF)

```

Comment:
Oh, that's pretty easy...

Reply:
DRFLAC is bundled so it'd be trivial to enable if needed

For OGG, we'd need to add a dependency on the ogg/vorbis libraries.
That's not difficult either.